controllers/get.js

```diff
@@ -0,0 +1,11 @@
+const {selectArticles} = require(`../models/selectArticles.js`);
+
+function getNewsData (req, res, next) {
+    selectArticles().then((articles) => {
+        res.status(200).send(articles);
+    }).catch((err) => {
```

Comment:
Good to have the catch and invocation of `next` here although at the moment you don't have an error handler in the app so it doesn't do anything at the moment.
However, you'll find that errors will only start to occur from clients in subsequent endpoints

Reply:
Added! I think.